lte/gateway/fabfile.py

```diff
@@ -253,13 +253,22 @@ def integ_test(gateway_host=None, test_host=None, trf_host=None,
     execute(_oai_coverage)
 
 
-def run_integ_tests():
+def run_integ_tests(tests=None):
     """
     Function is required to run tests only in pre-configured Jenkins env.
+    To Run:
+    make integ_test TESTS=s1aptests/test_attach_detach.py
+    
+    We need to run this command as:
+    fab run_integ_tests:tests=s1aptests/test_attach_detach.py
+
```

Comment:
maybe add a comment to indicate that default option (NONE) will run all tests

Reply:
thanks for the comment. just updated